### ISSUE TITLE
VLDB 2020: Industrial Track Papers

### ISSUE THREAD

**Issue by aaronchenwei** [aaronchenwei/awesome-industrial-papers#3]
https://vldb2020.org/accepted-papers.html

**Comment by aaronchenwei** [comment]
|               | Paper Title                                                                                                                                   |
| ------------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
| Facebook      | PyTorch Distributed: Experiences on Accelerating Data Parallel Training                                                                       |
| SAP           | Towards Multi-way Join Aware Optimizer in SAP HANA                                                                                            |
| Baqend        | InvaliDB: Scalable Push-Based Real-Time Queries on Top of Pull-Based Databases (Extended)                                                     |
| Oracle        | Automated Generation of Materialized Views in Oracle                                                                                          |
| Oracle        | Native JSON Datatype Support: Maturing SQL and NoSQL convergence in Oracle Database                                                           |
| PingCAP       | TiDB: A Raft-based HTAP Database                                                                                                              |
| Huawei        | A system design for elastically scaling transaction processing engines in virtualized servers                                                 |
| Huawei        | Industrial Strength OLTP Using Main Memory and Many Cores                                                                                     |
| SAP           | Asymmetric-Partition Replication for Highly Scalable Distributed Transaction Processing in Practice                                           |
| Ant Financial | AGL: A Scalable System for Industrial-purpose Graph Machine Learning                                                                          |
| Ant Financial | LedgerDB: A Centralized Ledger Database for Universal Audit and Verification                                                                  |
| Alibaba       | AnalyticDB-V: A Hybrid Analytical Engine Towards Query Fusion for Structured and Unstructured Data                                            |
| Oracle        | Oracle AutoML: A Fast and Predictive AutoML Pipeline                                                                                          |
| Google        | Monarch: Google’s Planet-Scale In-Memory Time Series Database                                                                                 |
| Microsoft     | Concurrent Updates to Pages with Fixed-Size rows Using Lock-Free Algorithms                                                                   |
| Microsoft     | POLARIS: The Distributed SQL Engine in Azure Synapse                                                                                          |
| Facebook      | MyRocks: LSM-Tree Database Storage Engine Serving Facebook’s Social Graph                                                                     |
| Microsoft     | Helios: Hyperscale Indexing for the Cloud & Edge                                                                                              |
| IBM           | Replication at the Speed of Change – a Fast, Scalable Replication Solution for Near Real-Time HTAP Processing                                 |
|               | Exploiting Domain Knowledge to address Multi-Class Imbalance and a Heterogeneous Feature Space in Classification Tasks for Manufacturing Data |
| Alibaba       | Alibaba Hologres: A Cloud-Native Service for Hybrid Serving/Analytical Processing                                                             |
| Google        | DIAMetrics: Benchmarking Query Engines at Scale                                                                                               |
| IBM           | Db2 Event Store: A Purpose-Built IoT Database Engine                                                                                          |
| Google        | F1 Lightning: HTAP as a Service                                                                                                               |
| Microsoft     | AutoToken: Predicting Peak Parallelism for Big Data Analytics at Microsoft                                                                    |
| ATT           | A Drop-in Middleware for Serializable DB Clustering across Geo-distributed Sites                                                              |
| IBM           | Improving Reproducibility of Data Science Pipelines through Transparent Provenance Capture                                                    |
| IBM           | Conversational BI: An Ontology-Driven ConversationSystem for Business Intelligence Applications                                               |
| LinkedIn      | Magnet: Push-based Shuffle Service for Large-scale Data Processing                                                                            |
| Google        | Leveraging Organizational Resources to Adapt Models to New Data Modalities                                                                    |
| Databricks    | Delta Lake: High-Performance ACID Table Storage over Cloud Object Stores                                                                      |

**Comment by aaronchenwei** [comment]
**Best Industrial Paper Award**
DIAMetrics: Benchmarking Query Engines at Scale

**Comment by aaronchenwei** [comment]
**Honorable Mention**
MyRocks: LSM-Tree Database Storage Engine Serving Facebook's Social Graph
